.doc_gen/metadata/firehose_metadata.yaml

```diff
@@ -0,0 +1,18 @@
+# zexi 0.4.0
+firehose_PutRecordBatch:
+  title: "&FHLong; batch record write"
+  title_abbrev: Writes multiple data records
+  synopsis: Writes multiple data records into a &FH; delivery stream in a single call, which can achieve higher throughput per producer than when writing single records.
```

Comment:
write multiple...
(lower case/verb conjugation because the synopsis has "The following code example shows how to" preprended to it).